guide/packages.md

```diff
@@ -0,0 +1,155 @@
+
+
+### Audio codecs
+
+| Package                           | Description                                         | Links                                                                                                                                                                                                    |
+| --------------------------------- | --------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `membrane_aac_plugin`             | AAC parser and complementary elements for AAC codec | [Hex](https://hex.pm/packages/membrane_aac_plugin) [Docs](https://hexdocs.pm/membrane_aac_plugin) [GitHub](https://github.com/membraneframework/membrane_aac_plugin)                                     |
+| `membrane_element_fdk_aac`        |                                                     | [Hex](https://hex.pm/packages/membrane_element_fdk_aac) [Docs](https://hexdocs.pm/membrane_element_fdk_aac) [GitHub](https://github.com/membraneframework/membrane-element-fdk-aac)                      |
+| `membrane_element_flac_parser`    |                                                     | [Hex](https://hex.pm/packages/membrane_element_flac_parser) [Docs](https://hexdocs.pm/membrane_element_flac_parser) [GitHub](https://github.com/membraneframework/membrane-element-flac-parser)          |
+| `membrane_element_lame`           |                                                     | [Hex](https://hex.pm/packages/membrane_element_lame) [Docs](https://hexdocs.pm/membrane_element_lame) [GitHub](https://github.com/membraneframework/membrane-element-lame)                               |
+| `membrane_element_mad`            |                                                     | [Hex](https://hex.pm/packages/membrane_element_mad) [Docs](https://hexdocs.pm/membrane_element_mad) [GitHub](https://github.com/membraneframework/membrane-element-mad)                                  |
+| `membrane_element_mpegaudioparse` |                                                     | [Hex](https://hex.pm/packages/membrane_element_mpegaudioparse) [Docs](https://hexdocs.pm/membrane_element_mpegaudioparse) [GitHub](https://github.com/membraneframework/membrane-element-mpegaudioparse) |
+| `membrane_opus_plugin`            | OPUS decoder                                        | [Hex](https://hex.pm/packages/membrane_opus_plugin) [Docs](https://hexdocs.pm/membrane_opus_plugin) [GitHub](https://github.com/membraneframework/membrane_opus_plugin)                                  |
+| `membrane_element_flac_encoder`   | [Suspended]                                         | [Hex](https://hex.pm/packages/membrane_element_flac_encoder) [Docs](https://hexdocs.pm/membrane_element_flac_encoder) [GitHub](https://github.com/membraneframework/membrane-element-flac-encoder)       |
```

Comment:
Flac encoder was not released (no hex & docs)

Reply:
master is empty, I'm not sure we should even list it

Reply:
I decided to list such stuff as suspended if someone wanted to contribute. I'll remove the master branch from that repo though